facebook.go

```diff
@@ -5,55 +5,78 @@ import "fmt"
 // FacebookRequest received from Facebook server on webhook, contains messages, delivery reports and/or postbacks
 type FacebookRequest struct {
 	Entry []struct {
-		ID        int64 `json:"id"`
+		ID        string `json:"id"`
 		Messaging []struct {
```

Comment:
Could you please create Messaging type out of FacebookRequest? I think it should be convenient for others to use Messaging type as an argument in thiers programms.